### PR TITLE
Fix console call error

### PR DESCRIPTION
**Pull request by wrefgtzweve**
- Fixes server console running the command returning errors
- Fixes no response when the player has no friends

copilot:poem

### CODE REVIEW

File: lua/ulx/modules/sh/cfc_checkfriends.lua [hunk 1]
```diff
@@ -26,6 +38,7 @@ net.Receive( "CFC_ULX_CheckFriendsReceive", function( _, ply )
 
     local caller = awaitingResponse[ply]
     if table.IsEmpty( friendTable ) then
+        caller:PrintMessage( 2, ply:Name() .. "does currently not have any friends on the server." )
```
Comment (legokidlogan) [inline lua/ulx/modules/sh/cfc_checkfriends.lua:41]: This should also perform a validity check on caller.

**Review by legokidlogan — CHANGES_REQUESTED** [review]

File: lua/ulx/modules/sh/cfc_checkfriends.lua [hunk 2]
```diff
@@ -26,6 +38,11 @@ net.Receive( "CFC_ULX_CheckFriendsReceive", function( _, ply )
 
     local caller = awaitingResponse[ply]
     if table.IsEmpty( friendTable ) then
+        if IsValid( caller ) then
+            caller:PrintMessage( 2, ply:Name() .. "does currently not have any friends on the server." )
+        else
+            print( ply:Name() .. "does currently not have any friends on the server." )
```
Comment (legokidlogan) [inline lua/ulx/modules/sh/cfc_checkfriends.lua:44]:
```suggestion
            caller:PrintMessage( 2, ply:Name() .. " does currently not have any friends on the server." )
        else
            print( ply:Name() .. " does currently not have any friends on the server." )
```

**Review by legokidlogan — CHANGES_REQUESTED** [review]

**Review by legokidlogan — APPROVED** [review]

**Comment by brandonsturgeon** [comment]
What does this do

**Comment by wrefgtzweve** [comment]
copilot:poem
test

**Pull request merged**